.env.sample

```diff
@@ -1,4 +1,3 @@
-APP_ID=asdfasdf
-APP_SECRET=secret-secret-secret
-REFRESH_TOKEN=ruXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
-NEXT_PUBLIC_TITLE=ARNAB
\ No newline at end of file
+APP_KEY=qy1yfn7mclvybv9
+APP_SECRET=dylisltrpw3r24a
```

Comment:
I hope this is just random string

Reply:
no it's there for a reason, so that user have to follow less steps